app/src/main/java/org/mozilla/focus/utils/Browsers.java

```diff
@@ -214,9 +214,10 @@ public boolean isInstalled(KnownBrowser browser) {
     /**
      * Is *this* application the default browser?
      */
-    public boolean isDefaultBrowser(Context context) {
-        return defaultBrowser != null && context.getPackageName().equals(defaultBrowser.packageName);
-
+    public static boolean isDefaultBrowser(Context context) {
+        final Browsers browser = new Browsers(context, "http://mozilla.org");
```

Comment:
Can we make this browser instance a singleton so we don't have to create it multiple times?

Reply:
If it worth to do then we can file a new issue to do it